packages/jest-cli/src/__tests__/cli/args.test.js

```diff
@@ -38,12 +38,6 @@ describe('check', () => {
     );
   });
 
-  it('sets onlyChanged if lastCommit is specified', () => {
```

Comment:
This test case is removed because it does not apply anymore; the logic has been moved to `normalize` in `jest-config`. Which, in turn, it's the right place to have it (`argv` should be read-only).

Reply:
Nice!